<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="customerList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Academy Engraved LET&quot;;
font: 20pt &quot;Nanum Gothic&quot;;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chatHistory">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>351</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chatConnectButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>채팅 접속</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>chatHistory</tabstop>
  <tabstop>customerList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
